telegram/helpers.py

```diff
@@ -55,9 +55,10 @@ def escape_markdown(text: str, version: int = 1, entity_type: str = None) -> str
             See the official API documentation for details. Only valid in combination with
             ``version=2``, will be ignored else.
     """
+    md_version_2 = 2
     if int(version) == 1:
         escape_chars = r"_*`["
-    elif int(version) == 2:
+    elif int(version) == md_version_2:
```

Comment:
see above. If ruff is complaining about hard-coded comparisons, why isn't it complaining about `== 1`?

Reply:
no idea, maybe because it's very common to check for hard-coded 0 and 1?